src/main/java/edu/ucsb/cs156/courses/controllers/CourseByInstructorController.java

```diff
@@ -20,11 +21,10 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 @RestController
+@Slf4j
```

Comment:
What's this for?

Reply:
This was for Seeing logs within the backend to see what was going wrong, I can remove it in my next commit.